main_window/packet_parser.py

```diff
@@ -198,20 +223,17 @@ def single_sensor(self, byte_stream: BytesIO, header: Header):
         :return:
         :rtype:
         """
-        subpacket_id = header.subpacket_id # TODO Review w/ https://trello.com/c/uFHtaN51 https://trello.com/c/bA3RuHUC
-        data = {}
-        # TODO Commented out, since apparently we pass along state, and other non-floats items, as float too??
-        # if subpacket_id == SubpacketEnum.STATE.value:
-        #     data[subpacket_id] = byte_list[0]
-        # else:
-        #     data[subpacket_id] = self.fourtofloat(byte_list[0])
-        data[subpacket_id] = self.fourtofloat(byte_stream.read(4))
+        # Transform int single sensor id to global DataEntryId
+        data_id = DataEntryIds[SubpacketIds(header.subpacket_id).name]
```

Comment:
Ooo I really like this!

Reply:
Pretty convenient we can use the '[]' and '()' for getting enum entries by name and value, respectively!